xbmc/cores/AudioEngine/AEDefines.h

```diff
@@ -22,5 +22,7 @@
 #define AE_AC3_ENCODE_BITRATE 640000
 #define AE_DTS_ENCODE_BITRATE 1411200
 
+#ifdef BUILDING_WITH_CMAKE
```

Comment:
put it in words: why should the code know if its built with cmake or whatever system else?